library/core/src/main/java/com/google/android/exoplayer2/ExoPlayerImpl.java

```diff
@@ -228,6 +243,31 @@ public boolean experimentalIsSleepingForOffload() {
     return playbackInfo.sleepingForOffload;
   }
 
+  @Override
```

Comment:
Similarly, can we move the methods to LivePlaybackSpeedControl?

Reply:
Sure will keep `getLiveConfiguration()` and remove everything else